mbtb_app/web_portal/mbtb_portal/assets/js/suspend_user_form.js

```diff
@@ -0,0 +1,19 @@
+
+  let requests_ids = get_checkbox_values();
+
+  let suspend_reason=document.getElementById("reason").value;
+
+  let post_request = $.post( "/suspend-user-with-reason", {requests_ids: requests_ids}, function(data, status) {
+    if (data === 'Success'){
+      // display alert on success and reload window
+      alert("Your selected user accounts are suspended.");
+      location.reload();
+    }
+    else {
+      alert("Something went wrong, Please try again.");
+    }
+
```

Comment:
4 -> remove lines 7 to 14 from here, we don't need validation here as we'll be passing data to form.